scripts/Auth_scripts/auth.ps1

```diff
@@ -0,0 +1,36 @@
+Function authUser($url)
+{
+    while($true)
+    {
+        $username = Read-Host 'Please enter your username'
+        $password = Read-Host 'Please enter your password' -AsSecureString
+        $body = @{
+            Username=$username

```

Comment:
Put the whitespace between the assigning symbol =

Reply:
Done